.devcontainer/Dockerfile

```diff
@@ -2,9 +2,9 @@
 
 # [Choice] .NET version: 5.0, 3.1, 2.1
 ARG VARIANT="5.0"
-FROM mcr.microsoft.com/vscode/devcontainers/dotnet:0-${VARIANT}
+FROM mcr.microsoft.com/vscode/devcontainers/dotnet
```

Comment:
seemed the `VARIANT` argument missed?

Reply:
This was intentionally since we want to update to whatever the latest .NET image is.

FWIW, the version in the image is irrelevant since we install our own for the repo.

Reply:
Then the argument could be  removed I think